src/command.cpp

```diff
@@ -76,8 +71,6 @@ struct {
     const char** list;
 } g_completions[] = {
```

Comment:
This is now effectively empty

Reply:
I've brought the content back, such that I can remove the unused code from `command.cpp` in a separate commit.